regression-test/suites/datatype_p0/nested_types/query/array_functions/test_array_functions_array_with_const.groovy

```diff
@@ -16,6 +16,7 @@
 // under the License.
 
 suite("test_array_functions_array_with_const", "p0") {
+    sql "set enable_nereids_planner=false;"
```

Comment:
why set to false? Is Nererids planner ok for this case?

Reply:
below in this groovy we have nereids query